universal-application-tool-0.0.1/app/auth/UatProfileData.java

```diff
@@ -20,7 +21,7 @@
   private Clock clock;
 
   public UatProfileData() {
```

Comment:
suggestion (non-blocking): At some point we should probably change this to an `@Inject` constructor that takes the `Clock` as a parameter, since it's already bound in the main module. That way we won't need to keep track of two clocks

Reply:
Sure - I don't understand how to call an `@Inject` constructor, though.  Can you make the change in a tiny PR and send it to me so I can see how it works?